src/index.js

```diff
@@ -26,4 +26,10 @@ naclFactory.instantiate(nacl => {
 	naclInstance = nacl;
 });
 
-export default { cryptography, transaction, API, time, passphrase };
+export default {
+	APIClient,
+	cryptography,
+	passphrase,
+	transaction,
+	time,
```

Comment:
`time` should already be part of `transaction.utils` - what we need here is exporting the `lisk-constants` module.